cupy/_math/sumprod.py

```diff
@@ -341,7 +342,7 @@ def gradient(f, *varargs, axis=None, edge_order=1):
     if axis is None:
         axes = tuple(range(ndim))
     else:
-        axes = internal._normalize_axis_indices(axis, ndim)
+        axes = nx.normalize_axis_tuple(axis, ndim)
```

Comment:
`internal._normalize_axis_indices(axis, ndim, sort_axes=False)` is also fine. Let's discuss we use which one in #4522.